src/main/java/com/project/voa/controller/IndexController.java

```diff
@@ -1,10 +1,10 @@
 package com.project.voa.controller;
 
-import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.servlet.ModelAndView;
 
-@Controller
+@RestController
 public class IndexController {
```

Comment:
Since the behavior of the issue is defined as a controller, how about changing the class name related to the issue?